src/components/Chart.js

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import axios from "axios";
+import Highcharts from "highcharts/highstock";
+import HighchartsReact from "highcharts-react-official";
+import { useState, useEffect } from "react";
```

Comment:
```suggestion
import React, { useState, useEffect } from "react";
import axios from "axios";
import Highcharts from "highcharts/highstock";
import HighchartsReact from "highcharts-react-official";
```
이렇게 작성하면 코드 줄을 조금 더 줄일 수 있을 거 같습니다 !